src/store/modules/ui.store.js

```diff
@@ -228,6 +233,26 @@ export default {
         noFeatureInfo(state) {
             return state.featureInfoPosition === FeatureInfoPositions.NONE
         },
+        /**
+         * Flag to display to display give feedback button/form. On localhost, it will always shown
+         * for testing purpose
+         */
+        hasGiveFeedbackButton(state, getters) {
+            if (getters.hasDevSiteWarning) {
+                return true
+            }
+            return GIVE_FEEDBACK_HOSTNAMES.some((hostname) => state.hostname.includes(hostname))
+        },
+        /**
+         * Flag to display to display report problem button/form. On localhost, it will always shown
+         * for testing purpose
+         */
+        hasReportProblemButton(state, getters) {
+            if (getters.hasDevSiteWarning) {
+                return true
+            }
+            return !getters.hasGiveFeedbackButton
```

Comment:
Based on other comment better to have a new config variable `REPORT_PROBLEM_HOSTNAMES`